prometheus_client/metrics.py

```diff
@@ -17,6 +18,9 @@
 from .samples import Exemplar, Sample
 from .utils import floatToGoString, INF
 
+if sys.version_info >= (3, 8, 0):
```

Comment:
Python 3.7 was just marked EOL a couple weeks ago, which reminded me to open https://github.com/prometheus/client_python/pull/936. What would you think of merging that first so we do not need this check?

Reply:
Sure! I noticed python 3.6 and 3.7 around the repository and decided not to break it.

Let's then wait for the other PR and I will happily rebase

Reply:
Support for 3.7 has been removed, so feel free to rebase this PR. Thank you!